From: Caleb Connolly <caleb.connolly@linaro.org>
Date: Fri, 13 Dec 2024 02:05:05 +0100
Subject: [PATCH] sdm845.config: ZRAM zstd and modules compression

Signed-off-by: Caleb Connolly <caleb.connolly@linaro.org>
(cherry picked from commit 88a2530cbd028bb6fd4b9dab2b0920ea01e9dee3)
Link: https://gitlab.com/sdm845-mainline/linux
Signed-off-by: Richard Acayan <mailingradian@gmail.com>
---
 arch/arm64/configs/sdm845.config | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/configs/sdm845.config b/arch/arm64/configs/sdm845.config
index 0fa43ef..9e7a9cf 100644
--- a/arch/arm64/configs/sdm845.config
+++ b/arch/arm64/configs/sdm845.config
@@ -337,6 +337,11 @@ CONFIG_CRYPTO_LZ4=m
 CONFIG_LZ4_COMPRESS=m
 CONFIG_CRYPTO_LZO=m
 CONFIG_CRYPTO_ZSTD=m
+CONFIG_ZRAM_BACKEND_ZSTD=y
+
+# module compression
+CONFIG_MODULE_COMPRESS=y
+CONFIG_MODULE_COMPRESS_ZSTD=y
 
 # pmOS iwd kconfig
 CONFIG_CRYPTO_USER_API_HASH=m
